### PR TITLE
- Fix api key

### PR DESCRIPTION
**Pull request by antsupovsa**

### CODE REVIEW

**Comment by antsupovsa** [comment]
Sorry, this request fixed config

**Comment by andreich1980** [comment]
Works perfect! Thank you!

File: src/Dangermark/GetResponse/GetResponse.php [hunk 1]
```diff
@@ -340,26 +342,26 @@ private function call($api_method = null, $http_method = 'GET', $params = array(
...
             CURLOPT_SSL_VERIFYPEER => 0
         );
 
-        if (!empty($this->enterprise_domain)) {
-            $options[CURLOPT_HTTPHEADER][] = 'X-Domain: ' . $this->enterprise_domain;
+        if (!empty($this->config('enterpriseDomain'))) {
+            $options[CURLOPT_HTTPHEADER][] = 'X-Domain: ' . $this->config('enterpriseDomain');
         }
 
         if (!empty($this->app_id)) {
```
Comment (andreich1980) [inline src/Dangermark/GetResponse/GetResponse.php]: Shouldn't here be `config('appId')` instead of `app_id`?

**Comment by antsupovsa** [comment]
@andreich1980 fixed,  did not look :(

**Pull request merged**